Index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>landing Page</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
```

Comment:
since we have that tag in HTML it's better to use another descriptive name for it